pkg/workflows/management/core_components.go

```diff
@@ -85,6 +90,40 @@ func runUpgradeCoreComponents(ctx context.Context, commandContext *task.CommandC
 	}
 	commandContext.UpgradeChangeDiff.Append(changeDiff)
 
+	client, err := commandContext.ClientFactory.BuildClientFromKubeconfig(commandContext.ManagementCluster.KubeconfigFile)
+	if err != nil {
+		commandContext.SetError(err)
+		return err
+	}
+
+	eksaCluster := &anywherev1.Cluster{}
+	err = client.Get(ctx, commandContext.CurrentClusterSpec.Cluster.Name, commandContext.CurrentClusterSpec.Cluster.Namespace, eksaCluster)
+	if err != nil {
+		commandContext.SetError(err)
+		return err
+	}
+
+	eksaCluster.SetManagementComponentsVersion(commandContext.ClusterSpec.EKSARelease.Spec.Version)
+
+	resourcesSpec, err := yaml.Marshal(eksaCluster)
+	if err != nil {
+		commandContext.SetError(err)
+		return err
+	}
+
+	logger.V(6).Info(string(resourcesSpec))
```

Comment:
do we need this? or was it for debugging

Reply:
thanks for catching, yeah it was for debugging